code/modules/vending/autodrobe.dm

```diff
@@ -122,7 +122,50 @@
 					/obj/item/clothing/suit/changshan_blue = 1,
 					/obj/item/clothing/suit/cheongsam_red = 1,
 					/obj/item/clothing/suit/cheongsam_blue = 1,
-					/obj/item/gohei = 1)
+					/obj/item/gohei = 1,
+					/obj/item/storage/backpack/henchmen = 20,
```

Comment:
not sure if there should be 20 of each of these, seeing as how every other costume in the vendor only has one or two copies of it in stock.

if you do want to keep high stocks of each of these costumes in each vending machine, I'd recommend increasing the stocks of the other costumes in these vending machines to match them.